### ISSUE TITLE
MobileNetV2 SSD Inference

### ISSUE THREAD

**Issue by oneTimePad** [tensorflow/models#3866]
I trained SSD with the MobileNetV2 and achieved a loss of around .8. However, when I run inference on it, all of the scores it outputs are very low. All are less .006. This is even true when it's ran on images from the training set. 

Has anyone else experienced this issue?

**Comment by oneTimePad** [comment]
I noticed a similar trend with embedded Mobilenet V1 SSD. Except in this case, all scores are very close together and approximately equal to .3.

**Comment by ahundt** [comment]
Can you post your command to run training for MobileNetV2? I'm still trying to piece together an example to test out the API

**Comment by oneTimePad** [comment]
@ahundt  Sorry for the late response:

Here is the command I used to train SSD MobileNetV2: 
`python3 object_detection/train.py --pipeline_config_dir $HOME/pipeline_config.config --train_dir $HOME/train_dir`

For inference, I utilized the object detection tutorial file.

**Comment by oneTimePad** [comment]
I still seem to be having a problem with this. I also noticed that it is not an issue with SSD mobilenetV2 but with  SSD mobilenetV1 as well. When I export older models (SSD mobilenetV1) that I have trained it works fine. However, any model trained from more recent commits, during inference, outputs all scores of around 0.

**Comment by FoilHao** [comment]
I have the same problem when implementing ssd_resnet50_fpn.
The training process seems fine but when testing, all the scores output are very weak and around 0.004.
I am trying to switch to ssd_mobilenetV1 right now.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
